feelpp/feel/feelmesh/bvh.hpp

```diff
@@ -343,7 +349,7 @@ namespace Feel
                                 std::vector<int> &orderedPrims)
         {
             LOG(INFO) <<fmt::format("cut dimension {}, start index primitive {}, end index primitive {}",cut_dimension,start_index_primitive,end_index_primitive);
-            Eigen::VectorXd M_bound_min_node(nDim),M_bound_max_node(nDim);
+            Eigen::VectorXd M_bound_min_node(realDim),M_bound_max_node(realDim);
```

Comment:
use static vector eigen instead of dynamic . This is much more efficient. And if possible, put in cache (i.e. only one allocation during the alog)

And remove the `M_` prefix, it's reserved for attribute member, not local variable.